projects/components/src/sharing-modal/tabs/sharing-modal-tab.component.scss

```diff
@@ -0,0 +1,116 @@
+::ng-deep .clr-form-control {
+    margin-top: 0 !important;
+}
+
+clr-datagrid {
```

Comment:
Why is this needed instead of just a height?

Reply:
These styles were no longer needed anymore, so I removed them.